src/freenet/client/filter/FilterUtils.java

```diff
@@ -383,9 +383,9 @@ public static boolean isColor(String value)
 			if(value.length()==4)
 			{
 				try{
-					Integer.valueOf(value.substring(1,2),16).intValue();
-					Integer.valueOf(value.substring(2,3),16).intValue();
-					Integer.valueOf(value.substring(3,4),16).intValue();
+                    Integer.valueOf(value.substring(1, 2), 16);
+                    Integer.valueOf(value.substring(2, 3), 16);
+                    Integer.valueOf(value.substring(3, 4), 16);
```

Comment:
here the indentation seems to have broken. Please keep it consistent with the surrounding indentation, even if that may seem strange. Consistency wins against beauty here, because we cannot just re-indent all files without breaking all pull-requests.

Reply:
Should be fixed in [b32d928]

Thank you